CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/panes/DataAccessPane.java

```diff
@@ -438,15 +444,28 @@ private String getExceptionDescription(Throwable e) {
         final HBox rabRegionExectueHBox = new HBox();
         rabRegionExectueHBox.setSpacing(4);
 
+        final HBox goBottomHBox = new HBox();
+        goBottomHBox.setAlignment(Pos.TOP_RIGHT);
+        goBottomHBox.setPadding(new Insets(10));
+        vbox.getChildren().add(goBottomHBox);
+
         final QualityControlAutoButton rab = Lookup.getDefault().lookup(QualityControlAutoButton.class);
+        final QualityControlAutoButton rabBottom = DefaultQualityControlAutoButton.defaultQualityControlAutoButton(rab);
```

Comment:
Is it possible you can instantiate the bottom one the same way as the top. If someone was to create a version other than the default, they would probably want that version on the bottom as well.

It might be that you can actually reuse the existing rather than creating a duplicate

Reply:
Sadly not, by reusing the same element it removes it from its originally set location. My first thought was to just use the existing versions of the buttons and place them at the bottom as well. This removed them from the top and put them at the bottom as that was the last statement compiled. Then I tried creating a new variable that referenced the other ones and this still did the same thing. Thats why it lead me to doing the recent changes. I was not able to create a normal copy of the existing rab variable.

Reply:
Ok. Does instantiating the bottom one the same way work (i.e. via the lookup)?

Reply:
if it did this:
        final QualityControlAutoButton rabBottom = Lookup.getDefault().lookup(QualityControlAutoButton.class);
It wasn't working. Doing the lookup they must still reference the same qualityControlAutoButton. So trying to set another variable to it would override the previous one being rab.

Reply:
Ok. An idea I have then is adding a copy function. You could make it an abstract function in `QualityControlAutoButton` and then override it in the default one. That should allow you to have a separate object of the same type which you can use